testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/MultiprocessOrchestration.cs

```diff
@@ -16,6 +16,11 @@ public class MultiprocessOrchestration
     private static List<Process> s_Processes = new List<Process>();
     private static int s_TotalProcessCounter = 0;
 
+    static MultiprocessOrchestration()
+    {
+        GetPathToMultiprocessDirectory();
```

Comment:
A "get" is usually idempotent, this should be called "Initialize" or something like that?

Reply:
that's a fair point. I shall rename it.

Reply:
https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1195/commits/59f7306dc56222c15fc7e05d95c0a65164ea55a9